src/commands/reset.js

```diff
@@ -67,10 +67,13 @@ class ResetCommand extends BaseCommand {
 
           const projectvolumeNames = await dockerCompose.getVolumeNames(config.toEnvs());
 
-          await projectvolumeNames
+          const volumeNames = projectvolumeNames
             .filter((volumeName) => !coreVolumeNames.includes(volumeName))
-            .map((volumeName) => `${composeProjectName}_${volumeName}`)
-            .forEach(async (volumeName) => docker.getVolume(volumeName).remove());
+            .map((volumeName) => `${composeProjectName}_${volumeName}`);
+
+          for (const volumeName of volumeNames) {
+            await (docker.getVolume(volumeName)).remove();
+          }
```

Comment:
You can do in map:
```js
const volumeRemovePromises = 
const volume  = docker.getVolume(`${composeProjectName}_${volumeName}`);
return volume.remove();
```

and resulting array of promises resolve like this:

```js
await Promise.all(volumeRemovePromises);
```

Reply:
I don't understand the syntax you are using, where is the map in there? Is this a function? Where does the returned object go? I was able to get it working using `Promise.all()` now but I'm not sure this is the result you were expecting.

Reply:
You already iterating through volume names and you can chain one more `map` there which will call `remove` and return promise, so the result variable will be an array of promises that you can put to `Promise.all`

Reply:
Fixed, I think. Can't believe this took me so long to understand.